Code/src/main/java/nl/utwente/group10/ui/components/ValueBlock.java

```diff
@@ -0,0 +1,32 @@
+package nl.utwente.group10.ui.components;
+
+import javafx.fxml.FXMLLoader;
+import javafx.scene.control.Label;
+import nl.utwente.ewi.caes.tactilefx.fxml.TactileBuilderFactory;
+
+import java.io.IOException;
+
+/**
+ * ValueBlock is an extension of Block that contains only a value
+ * and does not accept input of any kind.
+ * A single output source will be generated in order to connect
+ * a ValueBlock to another Block.
+ * 
+ * Extensions of ValueBlock should never accept inputs, if so desired
+ * Block should be extended instead.
+ */
+public class ValueBlock extends Block {
```

Comment:
Push/Pull behaviour will be modified once ui-connections is finished, suggestion of push behaviour has been removed from OutputBlock.